eventuate-tram-producer-jdbc/src/main/java/io/eventuate/tram/messaging/producer/jdbc/MessageProducerJdbcImpl.java

```diff
@@ -5,21 +5,25 @@
 import io.eventuate.common.jdbc.EventuateSchema;
 import io.eventuate.tram.messaging.common.Message;
 import io.eventuate.tram.messaging.producer.common.MessageProducerImplementation;
-import org.springframework.beans.factory.annotation.Autowired;
+
+import java.sql.SQLException;
```

Comment:
SQLException should not appear anywhere!

Reply:
I got it